### DISCUSSION TITLE
How to send a broadcast to a specific user

### DISCUSSION THREAD

**Discussion by akabutnicer** [supabase/supabase#22001] [Questions] [answered]
So I'm creating a chatting application, (yes, I can use socket.io for this but I'd much prefer supabase, if possible of course) and I would want to create public DMs, with every client which I have already made, and it's working well. However, in addition to me wanting private DMs, it's been hard figuring out how to work it. With the supabase broadcasting feature, (for instance channel.send()), how exactly would I do this?

**Comment by GaryAustin1** [comment, marked as answer]
I don't use broadcast, but you would likely only have the ability to have a "secret key" room/channel that both know.  
This is probably something more for postgres_changes on realtime as that can enforce RLS.

**Reply by akabutnicer** [reply]
Ah, thanks for the response. Yes, I believe sharing keys will work.
